### ISSUE TITLE
terraform 0.4.0 doesn't work in Intellij IDEA 2020.1.1

### ISSUE THREAD

**Issue by moleksyuk** [hashicorp/terraform-ls#200]
### Server Version

```
 ~/Workspace/terraform-ls-bug  ./terraform-ls --version
0.4.0
```

### Terraform Version

```
 ~/Workspace/terraform-ls-bug  terraform --version
Terraform v0.12.28
```

### Client Version

```
IntelliJ IDEA 2020.1.1 (Ultimate Edition)
Build #IU-201.7223.91, built on April 30, 2020
Licensed to Mykhailo Oleksiuk
Subscription is active until March 24, 2021
Runtime version: 11.0.6+8-b765.40 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.5
```

### Terraform Configuration Files

```hcl
no configs
```

### Log Output

File `/tmp/tf-exec-{{lsPid}}-{{args}}.log ` doesn't exist.

### Expected Behavior

Setup of terraform-ls has to work.

### Actual Behavior

Setup of terraform-ls doesn't to work.

### Steps to Reproduce

1. Created a new project.
2. Put main.tf file there.
```hlc
terraform {
  required_version = "0.12.28"

  required_providers {
    aws = "~> 2.49"
  }
}

provider "aws" {
  region = "eu-west-3"
}
```
3. Installed LSP plugin - https://plugins.jetbrains.com/plugin/10209-lsp-support
4. Downloaded the latest version of terraform-ls - https://releases.hashicorp.com/terraform-ls/0.4.0/terraform-ls_0.4.0_darwin_amd64.zip
5. Unzip and put binary file to the root of project.
6. Specified asbolute path to binary as `/Users/moleksyuk/Workspace/terraform-ls-bug/terraform-ls` and all related params accorting to https://github.com/hashicorp/terraform-ls/blob/master/docs/USAGE.md#intellij-ide
7. Press `Apply` and getting error: (See screenshot).
![Screenshot 2020-07-02 at 13 42 21](https://user-images.githubusercontent.com/793472/86349626-1ef2b800-bc6a-11ea-8994-792d93763f1c.png)

8. When I add a new *.tf file getting error about 'No root module'. (See screenshot)
![Screenshot 2020-07-02 at 13 42 56](https://user-images.githubusercontent.com/793472/86349623-1dc18b00-bc6a-11ea-9f51-7936b8654ff2.png)

**Comment by radeksimko** [comment]
Hi @moleksyuk 
The first screenshot leads me to thinking that this may have something to do with how the IJ plugin launches the server. It looks like this is already a known bug reported here: https://github.com/gtache/intellij-lsp/issues/147

The second one suggests that it eventually did launch it, because the error message comes from the language server.
It means that you likely need to run `terraform init` in a folder (root module) where you'd typically run it, so that plugins are available for the language server to pull data from. Have you tried that?

**Comment by moleksyuk** [comment]
Hi @radeksimko,

> The first screenshot leads me to thinking that this may have something to do with how the IJ plugin launches the server. It looks like this is already a known bug reported here: [gtache/intellij-lsp#147](https://github.com/gtache/intellij-lsp/issues/147)
👍  thanks

> The second one suggests that it eventually did launch it, because the error message comes from the language server.
> It means that you likely need to run `terraform init` in a folder (root module) where you'd typically run it, so that plugins are available for the language server to pull data from. Have you tried that?

Tried to run `terraform init` but couldn't reproduce it again due to issue with start server. Didn't know how it was launched yesterday 🤷‍♂️.

**Comment by radeksimko** [comment]
Unfortunately there isn't much the server can do when it comes to being launched. That will have to be addressed on the client side (i.e. in the [IntelliJ extension](https://github.com/gtache/intellij-lsp)) but I would be happy to look into any issues which happen _after_ the server starts.

I managed to launch it as [per instructions](https://github.com/hashicorp/terraform-ls/blob/master/docs/USAGE.md#intellij-ide) - I tried IntelliJ IDEA 2020.1 and language server `0.4.0` - both absolute or relative path. That said I did run into a bug, which I filed under https://github.com/hashicorp/terraform-ls/issues/203 - I wonder if that's what you're also experiencing?

> File /tmp/tf-exec-{{lsPid}}-{{args}}.log doesn't exist.

The file logging is not enabled by default, it needs to be enabled by adding `-log-file` flag, see the [Troubleshooting section](https://github.com/hashicorp/terraform-ls/blob/master/docs/TROUBLESHOOTING.md) for more.

Let us know if you manage to launch it and need further investigation/help, otherwise @hashibot will close this issue in 30 days automatically as it's not actionable without further details.
